content/_help/verify-your-identity/verify-your-address-by-mail._en.md

```diff
@@ -12,17 +12,17 @@ redirect_from:
 
 Identity verification is the process where you prove you are you - and not someone pretending to be you. One step in this process requires you to provide your primary phone number to verify your identity.
 
-If you cannot verify by phone, we will mail you a letter with a one-time code. You will use this code to finish verifying your identity and setting up your account.
+If you cannot verify by phone, we will mail you a letter with a verification code. You will use this code to finish verifying your identity and setting up your account.
 
-The letter is sent via the U.S. Postal Service three to seven business days after you request the code. The one-time code will expire 30 days after the date when you submit your request.
+The letter is sent via the U.S. Postal Service three to seven business days after you request the code. The verification code will expire 30 days after the date when you submit your request.
 
-If you do not receive your letter, or your one-time code expires before you finish the identity verification process, you can request a new one-time code.
+If you do not receive your letter, or your verification code expires before you finish the identity verification process, you can request a new verification code.
```

Comment:
I think this would be clearer as "... you can request a new letter." (Edit: like it says just below this)